### ISSUE TITLE
Images not rendering

### ISSUE THREAD

**Issue by arkuyucu** [Snowflake-Labs/sfquickstarts#1577]
**Describe the bug**
Images that are referred in guides doesnt appear in the guide web pages
**URL of where you see the bug**
http://localhost:8000/guide/cloudtrail_log_ingestion/index.html?index=..%2F..index#5
**To Reproduce**
I open the guide and the images doesnt appear

**Expected behavior**
Images that are referred in guides doesnt appear in the guide web pages
<img width="820" alt="Screen Shot 2024-09-13 at 22 03 36" src="https://github.com/user-attachments/assets/807b607c-5318-40d5-b798-5a7e18f47945">

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by iamontheinet** [comment]
Yes, this is a known issue related to Gulp 5.0 and we're looking into resolving it.

**Comment by sfc-gh-erhodes** [comment]
I'm also having this issue, thanks for reporting and acknowledging it. I was going crazy.

**Comment by nicdavidson** [comment]
So I stumbled into this issue as well, following the getting started instructions to the letter. @iamontheinet is correct in that this is a gulp version issue, my guess is an incompatibility with the required version of node (14 - [set here](https://github.com/nicdavidson/sfquickstarts/blob/master/site/.nvmrc), but I am not a node/front end guy so I am really just guessing after multiple hours of messing with it.

However, I was able to reliably resolve this issue by uninstalling and down grading gulp:

```
cd site

# Remove old versions
npm uninstall -g gulp
npm uninstall gulp

# Install correct versions
npm install -g gulp-cli@2.3.0
npm install gulp@4.0.2 --save-dev

# Clean and reinstall
rm -rf node_modules package-lock.json
rm -rf sfguides/dist/* sfguides/build/*
npm install

# Verify versions
gulp --version

# Try running the server
npm run serve
```

Not sure if this is the "right" way or not, but it worked, and I have image rendering working now.
